docs/user/security/securing-kibana.asciidoc

```diff
@@ -74,20 +73,21 @@ For more information, see <<xpack-security-session-management, Session managemen
 
 . Restart {kib}.
 
-. [[kibana-roles]]Choose an authentication mechanism and grant users the privileges they need to
-use {kib}.
+. Temporarily log into {kib} using the built-in `elastic` superuser so you can create new users and assign roles. If you are running {kib}
+locally, go to `https://localhost:5601` to view the login page.
 +
---
-For more information on Basic Authentication and additional methods of
-authenticating {kib} users, see <<kibana-authentication>>.
+NOTE: The password for the built-in `elastic` user is typically set as part of the security configuration process on {es}. For more
+information, see {ref}/built-in-users.html[Built-in users].
 
-To manage privileges, open the main menu, then click *Stack Management > Roles*.
+. [[kibana-roles]]Create roles and users to grant access to {kib}.
++
+--
+To manage privileges in {kib}, open the main menu, then click *Stack Management > Roles*. The built-in `kibana_admin` role will grant
+access to {kib} with administrator privileges. Alternatively, you can create additional roles that grant more limited access to {kib}.
```

Comment:
nit: I understand what you're saying here, but "more limited access" initially tripped me up. I think we can simplify this by removing `more`:

```suggestion
access to {kib} with administrator privileges. Alternatively, you can create additional roles that grant limited access to {kib}.
```

Reply:
Fixed in 11c26855278.